guillotina/db/transaction.py

```diff
@@ -61,6 +62,8 @@ def __call__(self, func):
                 try:
                     if not this.check_state_size or len(result["state"]) < self._cache.max_cache_record_size:
                         await self._cache.set(result, **key_args)
+                        for key_gen in this.additional_keys:
+                            await self._cache.set(result, **key_gen(result))
```

Comment:
Caching an object with one additional key, will count the cache as using `2*size_of(obj)` ?

Reply:
Therefore, reducing the effective cache size by two?

Reply:
You're right. I'll need to think about this differently.